src/views/onboarding/collection-view/index.tsx

```diff
@@ -15,9 +15,6 @@ export default function OnboardingCollectionView({
 	metadata,
 }: OnboardingCollectionViewProps) {
 	const { name, id, description, tutorials, ordered, slug } = collection
-	const startTutorialSlug = `/${slug}/${splitProductFromFilename(
```

Comment:
Note: the relatively "special treatment" for Onboarding and WAF is handled in `getTutorialSlug`, so no further changes were needed to get things working 🙌

Reply:
Ah good catch, ty 😄